src/beta/resources/referral.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-dupe-class-members,no-unused-vars,class-methods-use-this */
```

Comment:
Remove this comment

Reply:
This is to stop the linter from aggressively complaining about linting inconsistencies that we don't care about.